service/surf/settings/base.py

```diff
@@ -377,3 +377,8 @@ def strip_sensitive_data(event, hint):
     DEBUG_TOOLBAR_CONFIG = {
         "SHOW_TOOLBAR_CALLBACK": lambda request: request.get_host() in INTERNAL_HOSTS
     }
+
+EMAIL_HOST = 'outgoing.surfmailfilter.nl'
+EMAIL_PORT = 25
```

Comment:
You guys were already talking about this before. So probably you already tried this, but wanted to give a heads up that secure email port is 587 and not 25.